lib/elixir/lib/exception.ex

```diff
@@ -997,6 +1017,20 @@ defmodule UndefinedFunctionError do
     ["      * ", Code.Identifier.inspect_as_function(fun), ?/, Integer.to_string(arity), ?\n]
   end
 
+  defp behaviours_for(module) do
+    attributes =
+      case function_exported?(module, :__info__, 1) do
+        true -> module.__info__(:attributes)
+        false -> module.module_info(:attributes)
+      end
+
+    Keyword.get(attributes, :behaviour, [])
+  rescue
```

Comment:
I would move this rescue to `behaviour_hint` since there are other operations there that may fail.